src/library/scala/util/Try.scala

```diff
@@ -141,6 +141,61 @@ sealed abstract class Try[+T] extends Product with Serializable {
     def withFilter(q: T => Boolean): WithFilter = new WithFilter(x => p(x) && q(x))
   }
 
+  /** Tests whether this is a `Success` with a given value.
+   *
+   * This is equivalent to:
```

Comment:
Just to note that if you want the text to align, there should be two spaces after the star. The community doesn't enforce self-consistent comment style, however. I think they call this Adriaan style.